tests/launch_tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Or I can create commit changing them to executables if it's too much trouble. You can cherry pick my commits from my branch. Whichever you prefer :)